### PR TITLE
fix for OctreeSceneStat unpacking in Application

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9c57d1544fdc237fd4476972795e91050cdffa86 available at: https://gist.github.com/f048946ae4c6c3849599<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.dmg

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -3596,23 +3596,17 @@ int Application::processOctreeStats(NLPacket& packet, SharedNodePointer sendingN
...
-    }
+    
+    OctreeSceneStats& octreeStats = _octreeServerSceneStats[nodeUUID];
+    statsMessageLength = octreeStats.unpackFromPacket(packet);
+    
     _octreeSceneStatsLock.unlock();
 
     VoxelPositionSize rootDetails;
-    voxelDetailsForCode(octreeStats->getJurisdictionRoot(), rootDetails);
+    voxelDetailsForCode(octreeStats.getJurisdictionRoot(), rootDetails);
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: it looks like this rootDetails is actually only used for debug log message in the case that it's a new server... there's a chance to optimize/clean this up by moving this code into the if. Not a merge blocker.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 615218c77dc7c809a01da4265159805a81101da5 available at: https://gist.github.com/f048946ae4c6c3849599<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d3c6d8b3ccd528297de8f7e517499be4171105ef available at: https://gist.github.com/f048946ae4c6c3849599<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.dmg

**Comment by birarda** [comment]
error seems pretty odd

```
CMake Error at /usr/share/cmake-3.2/Modules/FindPackageHandleStandardArgs.cmake:138 (message):
  Could NOT find OpenGL (missing: OPENGL_gl_LIBRARY)
Call Stack (most recent call first):
  /usr/share/cmake-3.2/Modules/FindPackageHandleStandardArgs.cmake:374 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.2/Modules/FindOpenGL.cmake:168 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  libraries/gpu/CMakeLists.txt:34 (find_package)
```

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: d3c6d8b3ccd528297de8f7e517499be4171105ef available at: https://gist.github.com/f048946ae4c6c3849599<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: d3c6d8b3ccd528297de8f7e517499be4171105ef available at: https://gist.github.com/f048946ae4c6c3849599<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5434.dmg

**Comment by ZappoMan** [comment]
I will test on mac and windows and then merge assuming it passes a basic smoke test.

**Pull request merged**